display/ssd1306.h

```diff
@@ -16,6 +16,12 @@ IMAGE_FILESET(blst);
 IMAGE_FILESET(lock);
 IMAGE_FILESET(force);
 IMAGE_FILESET(idle);
+IMAGE_FILESET(reload);
```

Comment:
I'm trying to decide the best way to have these not use any space for props that have displays, but are not guns.

Right now, the best I can come up with is to use defines, which gets a little ugly.
Basically we would want a set of defines which specifies which EFFECT_** entries to use.
For each EFFECT_** entry, we would also add a define HAVE_EFFECT_**, which can be use to #ifdef filesets and the case statements.

To make it easier to manage the defines, I was thinking that I could add sections to the prop files, similar to how the config files have sections. Then I could include the prop file early in the compilation progress, and let the prop file help define which EFFECT_** we need.

Not exactly pretty, but it could work.